packages/analytics/client/src/analytics_client/analytics_client.ts

```diff
@@ -43,7 +43,7 @@ import { ContextService } from './context_service';
 import { schemaToIoTs, validateSchema } from '../schema/validation';
 
 interface EventDebugLogMeta extends LogMeta {
-  ebt_event: Event;
+  ebt_event: Event<unknown>;
```

Comment:
Fixed the issue with the properties being forced to extend `Record<string, unknown>`